src/ptp.rs

```diff
@@ -110,9 +112,47 @@ pub trait PtpCommandCode: Sized + Copy {
             *a as u16
         }
     }
+
+    fn enum_name(&self) -> String;
 }
 
-impl PtpCommandCode for StandardCommandCode {}
+impl PtpCommandCode for StandardCommandCode {
+    fn enum_name(&self) -> String {
+        match *self {
```

Comment:
core::fmt::Debug isn't implemented for Enum, which is why this is required. Plus, then Rust would have to allocate memory for all of the enumeration names, which seems crazy to do by default.